### DISCUSSION TITLE
Long lag when entering vi mode

### DISCUSSION THREAD

**Discussion by ldelossa** [jeffreytse/zsh-vi-mode#275] [Q&A]
Hello, 

I'm experiencing a long lag when entering VI node. Not sure why. Is there a delay timer for Ctrl-[ (what I use)

**Comment by Ryoschin** [comment]
Yup, having the same issue with both `<esc>` and `<ctrl-[>`. Not sure if there's a keybind that uses `<esc>` or something like that, which would probably explain it taking 0.4 seconds to enter Normal mode. Using `dw`, `cw`, or anything related summons an abnormal delay as well...
